blocks/cards/cards.js

```diff
@@ -1,18 +1,17 @@
-import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
-
 export default function decorate(block) {
-  /* change to ul, li */
-  const ul = document.createElement('ul');
   [...block.children].forEach((row) => {
-    const li = document.createElement('li');
-    li.innerHTML = row.innerHTML;
-    [...li.children].forEach((div) => {
-      if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+    row.className = 'cards-item';
+    [...row.children].forEach((div) => {
+      if (div.querySelector('picture')) {
+        // update container for picture with label
+        div.className = 'cards-card-image';
+        if (div.lastChild.nodeType === Node.TEXT_NODE) {
+          const picture = div.querySelector('picture');
+          const paragraphElement = document.createElement(('p'));
+          paragraphElement.append(div.lastChild);
+          div.append(picture, paragraphElement);
+        }
+      } else div.className = 'cards-card-body';
```

Comment:
If i understand this correctly. The Label for the picture will be whatever text is on the left side of the card block, under the picture. Correct? 

Are we sure that there isn't already a paragraph tag made?

Reply:
This is a good call. [Here is an example](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B564955A9-FFFC-4BE0-A6AF-218836CC0451%7D&file=cards.docx&action=default&mobileredirect=true) of how we expect the author to add a label. We can add as a note always add a label from the new line to avoid this transformation